ReadME.md

```diff
@@ -0,0 +1,98 @@
+
+**1. How to start as a beginner so as to contribute to the project?**
+*There are many beginner level issues to begin with, which include documentation, development and design. Also, feel free to make issues for features as well as for reporting bugs. Participants can claim an issue by commenting that they are interested in it. We are also open to new ideas and suggestions.*
+
+**2. How to communicate with us?**
+*All communication will happen on the Slack channel for this particular projects. Mentors can also be reached through GitHub. Avoid DMs and unnecessary mentions on slack and GitHub both. *Checkout [this](https://github.com/CatsInTech/Rezume/issues) issue for more.**
+
+**3. What is the deadline for any task submission?**
+
+*Mentors can assign issues to participants and also mention a deadline before which the issue is to be resolved. They can reassign issues in case of any delay, so be active and complete your tasks within the negotiated duration.*
+
+## Mentors
+
+ [![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/CatsInTech/Rezume) ![GitHub pull requests](https://img.shields.io/github/issues-pr-closed-raw/CatsInTech/Rezume?logo=git&logoColor=white)
+
+| GitHub Usernames                                     | Domain                     |
+| ---------------------------------------------------- | -------------------------- |
+| [@praveenscience](https://github.com/praveenscience) | Full Stack MERN / LAMP + Documentation |
+| [@priyadharshinir1894](https://github.com/priyadharshinir1894) | Full Stack + Cloud         |
+| [@vermakhushboo](https://github.com/vermakhushboo) | Full Stack         |
+| [@PrathibhaS](https://github.com/PrathibhaS) | Backend |
+| [@AnjaliSharma1234](https://github.com/AnjaliSharma1234) | Frontend + Documentation |
+
+## Project Admin
+
+[![Relative date](https://img.shields.io/date/1577392258?color=important&label=started&logo=github)](https://github.com/praveenscience/) [![Maintenance](https://img.shields.io/maintenance/yes/2020?color=green&logo=github)](https://github.com/praveenscience/)
+
+
+
+| <a href="https://praveen.science/"><img src="https://i.imgur.com/pQkeC0B.png" width=150px height=120px /></a> |
+| :----------------------------------------------------------: |
+| **[Praveen Kumar Purushothaman](https://praveen.science/)**  |
+
+> ***Need help?***
+***Feel free to contact me at [catsintech@praveen.science](mailto: catsintech@praveen.science?Subject=Cats in Tech Project)***
+
+[![GitHub followers](https://img.shields.io/github/followers/praveenscience.svg?label=Follow%20@praveenscience&style=social)](https://github.com/praveenscience/) [![Twitter Follow](https://img.shields.io/twitter/follow/praveenscience?style=social)](https://twitter.com/praveenscience) 
+
+---
+
+[![built with love](https://forthebadge.com/images/badges/built-with-love.svg)](https://github.com/AnjaliSharma1234/)
+
+ 
```

Comment:
Get rid of trailing spaces on each lines and trailing lines at the end of document.